cw_bitcoin_cash/lib/src/bitcoin_cash_wallet.dart

```diff
@@ -83,7 +83,7 @@ abstract class BitcoinCashWalletBase extends ElectrumWallet with Store {
       unspentCoinsInfo: unspentCoinsInfo,
       initialAddresses: initialAddresses,
       initialBalance: initialBalance,
-      seedBytes: await MnemonicBip39.toSeed(mnemonic, passphrase: passphrase),
+      seedBytes: MnemonicBip39.toSeed(mnemonic, passphrase: passphrase),
```

Comment:
You are right, the `await` here is not needed. But now `static Future<BitcoinCashWallet> create(` can be `static BitcoinCashWallet create(` or am I wrong?

Reply:
You are right but it's minor can be skipped for now
Some code enhancements needs to be done in this flow anyway but I said I would do it in a separate pr better